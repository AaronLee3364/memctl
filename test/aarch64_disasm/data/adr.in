fffffff0070c0318 10af9c00 f00025c8
