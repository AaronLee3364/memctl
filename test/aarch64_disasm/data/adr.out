fffffff0070c0318  ADR     X0, #0xfffffff00701f698
fffffff0070c031c  ADRP    X8, #0xfffffff00757b000
